samples/snippets/csharp/VS_Snippets_CLR/DPAPI-HowTO/cs/sample.cs

```diff
@@ -101,10 +101,10 @@ public static void EncryptInMemoryData(byte[] Buffer, MemoryProtectionScope Scop
 
     public static void DecryptInMemoryData(byte[] Buffer, MemoryProtectionScope Scope)
     {
-        if (Buffer.Length <= 0)
```

Comment:
@laedit, thank you so much both for correcting the typo in the variable name, as well as reversing the order of the checks for a null array and an empty array! There's an additional check that should be reversed, though -- in lines 91-94, the array length is also checked before the test for null. Would you mind changing that code as well? If you'd prefer, we can also do it for you. Again, thank you so much for making these corrections.

Reply:
Thanks, I've missed that. I will do it as well.